neard/src/config.rs

```diff
@@ -550,6 +552,7 @@ pub struct NearConfig {
     pub client_config: ClientConfig,
     pub network_config: NetworkConfig,
     pub rpc_config: RpcConfig,
+    pub cache_config: CacheConfig,
```

Comment:
This is a too general name for this context. Ie, it's not clear it cache applies to chain/network/vm. `VMCacheConfig` would be better.

As an aside, it actually should be `VmCacheConfig`, according to Rust naming rules, but we have `VM` everywhere, so I guess its better to roll with that.

Reply:
Sounds good

Reply:
renamed